src/bin/sol-fbp-generator/main.c

```diff
@@ -1886,6 +1886,9 @@ print_usage(const char *program)
         "    -j  When resolving types, use the passed DESC files. If DESC is\n"
         "        a directory then all the .json files in the directory will be used.\n"
         "        Multiple -j can be passed.\n"
+        "        Note that Soletta install dir will also be looked, as well as any\n"
```

Comment:
I'd go with:
 - no envvar, look into install dir. This allows simple usage for the common case.
 - envvar is set, do not look into install dir. Even if envvar is set to an invalid directory. This allows easy cross compiling, just set the envvar to the cross-compiled directory, or set to some invalid/empty directory.
 - `-j` adds to/extends the above.